shell/plugins/steve/getters.js

```diff
@@ -85,7 +95,7 @@ export default {
     // `namespaced` is either
     // - a string representing a single namespace - add restriction to the url
     // - an array of namespaces or projects - add restriction as a param
-    if (opt.namespaced && !pAndNFiltering.isApplicable(opt)) {
+    if (opt?.namespaced && !pAndNFiltering.isApplicable(opt)) {
```

Comment:
Since a default object is not provided in the function, it's possible to call this function without providing the parameter and doing so would cause this function to crash. This implementation is less brittle.

Reply:
There is no other half of this if block, moreover, if opt were undefined then the previous code would crash as javascript would not be able to read the property `namespaced` of undefined so it wouldn't even get to the other half of the if block.